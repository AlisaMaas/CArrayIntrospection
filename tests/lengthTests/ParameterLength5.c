#include <stdio.h>

/*
* Tests a loop where one argument is length and
* one is an element to find. The length of x 
* should be y. More tricky than #3 because
* the length check is in a call to an 
* external function.
*
* Example contributed by Peter Ohmann.
*/
void print(char* x, int y){
  int i;
  for(i = 0; i < y; i++)
    printf("%c", x[i]);
}