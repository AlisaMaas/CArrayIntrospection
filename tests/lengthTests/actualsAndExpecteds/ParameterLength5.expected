Analyzing print
	Argument x has max index argument y
	Argument y has unknown max index.
Analyzing printf
	Argument  has unknown max index.
